commitizen/defaults.py

```diff
@@ -65,6 +67,7 @@ class Settings(TypedDict, total=False):
     "update_changelog_on_bump": False,
     "use_shortcuts": False,
     "major_version_zero": False,
+    "encoding": "utf-8",
```

Comment:
shouldn't this be

```py
"encoding": encoding,
```

Reply:
Good point. Should we also change

```python
"name": "cz_conventional_commits"
```

to

```python
"name": name
```

Reply:
Good point. Should we also change

```python
"name": "cz_conventional_commits"
```

to

```python
"name": name
```